upf/upf.c

```diff
@@ -382,6 +389,12 @@ upf_init (vlib_main_t * vm)
   sm->upf_simple_counters[UPF_FLOW_COUNTER].stat_segment_name =
     "/upf/total_flows";
 
+  sm->node_id.type = NID_FQDN;
+  u8 *name = 0;
+  name = format (name, (char *) "upg");
+  sm->node_id.fqdn = vec_dup (name);
+  vec_free (name);
```

Comment:
Are these `vec_dup` and `vec_free` really needed? `format()` returns a new vector. (it is indeed sometimes used to make a vector out of C string)

Reply:
Missed that part
Done